server.js

```diff
@@ -66,7 +66,10 @@ app.use(helmet.contentSecurityPolicy({
   },
 }));
 app.use(helmet.referrerPolicy({ policy: "strict-origin-when-cross-origin" }));
-app.use(express.static("public"));
+app.use(express.static("public", {
+  setHeaders: res => res.set("Cache-Control",
+    "public, maxage=" + 10 * 60 * 1000 + ", s-maxage=" + 24 * 60 * 60 * 1000),
```

Comment:
maxage is... 10 minutes and s-maxage is 24 hours?

Also, string above says "s-maxage" whereas comment below says "x-maxage". Not sure which is possible typo.

Reply:
Updated comment below.

`maxage` > 0 is needed so nginx will actually use `ETag|Last-Modified` and `If-None-Match|If-Modified-Since` values for cache negotiation. But I'm worried that a long client-side cache age could make assets stick, so I'm starting with a lower client-side cache max age. (Pretty much enough to keep the assets cached while a user is performing a few scans and signing up.)

`s-maxage` will make the CDN and/or nginx refresh the statics from upstream at least every 24 hours.